src-vue/src/views/ModsView.vue

```diff
@@ -1,15 +1,50 @@
 <template>
     <div class="fc__mods__container">
         <h3>Installed Mods:</h3>
-        TODO
+        <div>
+            <el-card shadow="hover" v-for="mod in installed_mods">
+                <el-switch style="--el-switch-on-color: #13ce66; --el-switch-off-color: #8957e5" v-model="mod.enabled"
+                    disabled />
+                {{mod.name}}
+            </el-card>
+        </div>
     </div>
 </template>
 
 <script lang="ts">
 import { defineComponent } from "vue";
+import { ElNotification } from "element-plus";
+import { invoke } from '@tauri-apps/api/tauri';
+import { GameInstall } from "../utils/GameInstall";
+import { NorthstarMod } from "../utils/NorthstarMod"
 
 export default defineComponent({
     name: "ModsView",
+    data() {
+        return {
+            installed_mods: [] as NorthstarMod[],
+        }
+    },
+    async mounted() {
+        let game_install = {
+            game_path: this.$store.state.game_path,
+            install_type: this.$store.state.install_type
+        } as GameInstall;
+        // Call back-end for installed mods
+        await invoke("get_installed_mods_caller", { gameInstall: game_install })
+            .then((message) => {
+                this.installed_mods = (message as NorthstarMod[]);
+            })
+            .catch((error) => {
+                console.error(error);
+                ElNotification({
```

Comment:
It's not very cool to directly display errors to end-users.
What errors do you expect here?

Reply:
Error is basically whatever the backend returns. Showing the backend error message unprocessed makes sense cause this allows us to show a detailed error message depending on the error (i.e. missing `enabledmods.json` vs incorrectly structured`enabledmods.json`).


Examples:

`enabledmods.json`  doesn't exist:

![image](https://user-images.githubusercontent.com/40122905/197606328-3c914dfe-aca6-436a-a760-80bc47c040d7.png)

Invalid `enabledmods.json`:

![image](https://user-images.githubusercontent.com/40122905/197606504-963cc355-2c9e-4f6f-b394-e3dbb76cd040.png)


Of course we could (should?) adjust the message that the backend actually sends but I'd do that in a separate PR once I also look into removing all the `.expect`s and `.unwrap`s. ^^